eventlogger/src/main/java/gov/cms/ab2d/eventlogger/ApiRequestEventLogger.java

```diff
@@ -0,0 +1,29 @@
+package gov.cms.ab2d.eventlogger;
+
+import lombok.Data;
+
+import java.time.OffsetDateTime;
+
+/**
+ * Class to create and log an API request coming from a user
+ */
+@Data
+public class ApiRequestEventLogger implements LoggableEvent {
```

Comment:
So the interface is called `event` while the implementation classes are called `loggers`, which might sound a bit awkward, since loggers aren't events. Should we drop the `Logger` post-fix from the class names?

Reply:
Sure